roles/container-engine/docker/molecule/default/tests/test_default.py

```diff
@@ -14,6 +14,6 @@ def test_docker_service(host):
 
 def test_docker_run(host):
     with host.sudo():
-        cmd = host.command("docker run hello-world")
+        cmd = host.command("docker run quay.io/aptible/hello-world:latest")
```

Comment:
@floryut could you cache the original image from docker hub in the kubespray namespace on quay.io? Somehow using this random image I found on quay.io doesn't feel that secure.

Reply:
I'll mirror it no problem 😊

Reply:
-> quay.io/kubespray/hello-world

Reply:
@cristicalin we could also mirror socat